spec.html

```diff
@@ -7114,10 +7114,10 @@ <h1>DisposableResource Records</h1>
       </emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-adddisposableresource-disposable-v-hint-disposemethod" type="abstract operation">
+    <emu-clause id="sec-adddisposableresource" type="abstract operation">
       <h1>
         AddDisposableResource (
-          _disposable_: an object with a [[DisposableResourceStack]] internal slot,
+          _disposable_: an Object that has a [[DisposableResourceStack]] internal slot,
```

Comment:
It's possible neither version is correct. This can either be an ECMAScript Object or a declarative Environment Record.

Reply:
Another option would be to have both the environment record and the `DisposableStack` object point to another Record type that holds the stack, and passes that Record to each function.

Reply:
That seems like a simpler model to me, good idea